sys/vm/vm_page.h

```diff
@@ -445,6 +463,7 @@ extern struct mtx_padalign pa_lock[];
 #define	PGA_NOSYNC	0x0080		/* do not collect for syncer */
 #define	PGA_SWAP_FREE	0x0100		/* page with swap space was dirtied */
 #define	PGA_SWAP_SPACE	0x0200		/* page has allocated swap space */
+#define	PGA_CAPDIRTY	0x0400		/* page targeted by cap store */
```

Comment:
Why is this not a `vm_page_bits_t capdirty;` in `struct vm_page` like `dirty` is?

Reply:
I suppose, in principle, we could track dirty bits for sub-page nature, but we don't.  Does anyone use the multiple bits in `dirty` as anything other than all zeros or all ones?

In earlier designs, the atomic nature of `PGA_CAPDIRTY` was more important than it has turned out to be but the bit never moved; it may already be sufficiently protected by the page's busy lock that the conversion is straightforward, but I'd have to check.